modules/svg/image_loader_svg.cpp

```diff
@@ -31,7 +31,6 @@
 
 #include "os/os.h"
 #include "print_string.h"
-
```

Comment:
Don't remove this, it's intended that Godot headers (`"name.h"`) are separate from system headers (`<name.h>`).